jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -73,6 +74,53 @@ local excludedRules = [
       { alert: 'PrometheusErrorSendingAlertsToAnyAlertmanager' },
     ],
   },
+  {
+    name: 'kube-prometheus-node-recording.rules',
+    rules: [
+      { alert: 'instance:node_cpu:rate:sum' },
+      { alert: 'instance:node_network_receive_bytes:rate:sum' },
+      { alert: 'instance:node_cpu:ratio' },
+      { alert: 'instance:node_network_transmit_bytes:rate:sum' },
+    ],
+  },
+  {
+    name: 'node.rules',
+    rules: [
+      { record: 'node:node_num_cpu:sum' },
+    ],
+  },
+  {
+    name: 'openshift-build.rules',
+    rules: [
+      { record: 'build_error_rate' },
+    ],
+  },
+  {
+    name: 'openshift-kubernetes.rules',
+    rules: [
+      { record: 'namespace:container_cpu_usage:sum' },
+      { record: 'namespace:container_memory_usage_bytes:sum' },
+      { record: 'namespace:container_spec_cpu_shares:sum' },
+      { record: 'pod:container_cpu_usage:sum' },
+      { record: 'pod:container_fs_usage_bytes:sum' },
+      { record: 'pod:container_memory_usage_bytes:sum' },
+      { record: 'pod:container_spec_cpu_shares:sum' },
```

Comment:
a couple of these rules are used by the [console](https://github.com/openshift/console)

Reply:
the following rules are referenced in console codes:
1. namespace:container_cpu_usage:sum
2. pod:container_cpu_usage:sum
3. pod:container_fs_usage_bytes:sum

These rules are not referenced and safe to remove:
1. namespace:container_spec_cpu_shares:sum
2. namespace:container_spec_cpu_shares:sum
3. pod:container_memory_usage_bytes:sum
4. pod:container_spec_cpu_shares:sum